auth-web/src/components/auth/manage-business/CreateNewBusinessHelp.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div id="help-resolution" class="mb-12">
+    <div class="ma-6">
+      <div class="help-btn top" @click="helpToggle = !helpToggle">
+        <v-icon color="primary">mdi-help-circle-outline</v-icon>
+        <span v-if="!helpToggle" class="pl-2">{{ helpContent.toggleHelpShow }}</span>
+        <span v-else class="pl-2">{{ helpContent.toggleHelpHide }}</span>
+      </div>
+      <v-expand-transition v-if="helpToggle">
```

Comment:
Weird. It contracts using a transition but it doesn't seem to expand using one. What do you see when you run this locally?

Try to put the v-if on the section inside this block.

Reply:
It's working correctly now.